api/src/certification/shared/infrastructure/repositories/certification-course-repository.js

```diff
@@ -16,43 +13,39 @@ import {
 import { NotFoundError } from '../../../../../lib/domain/errors.js';
 import * as certificationChallengeRepository from './certification-challenge-repository.js';
 import { CertificationIssueReport } from '../../domain/models/CertificationIssueReport.js';
-import { Bookshelf } from '../../../../../lib/infrastructure/bookshelf.js';
 
 async function save({ certificationCourse, domainTransaction = DomainTransaction.emptyTransaction() }) {
-  const knexConn = domainTransaction.knexTransaction || Bookshelf.knex;
+  const knexConn = domainTransaction.knexTransaction || knex;
+
   const certificationCourseToSaveDTO = _adaptModelToDb(certificationCourse);
-  const options = { transacting: domainTransaction.knexTransaction };
-  const savedCertificationCourseDTO = await new BookshelfCertificationCourse(certificationCourseToSaveDTO).save(
-    null,
-    options,
-  );
+  const [{ id: certificationCourseId }] = await knexConn('certification-courses').insert(certificationCourseToSaveDTO, [
+    'id',
+  ]);
```

Comment:
On peut optimiser un chouia
```suggestion
  const [{ id: certificationCourseId }] = await knexConn('certification-courses').insert(certificationCourseToSaveDTO, [
    'id',
  ]).returning('id');
```